pyproject.toml

```
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools>=65.5.1", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "linpyk"
+version = "0.1.0"
+description = "SDK for linear algebra on ArmoniK"
+readme = "README.md"
+authors = [{ name = "Quentin Delamea", email = "qdelamea@aneo.fr" }]
+classifiers = [
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+]
+keywords = ["linear algebra", "armonik", "graph", "task"]
+dependencies = [
+    "armonik >= 3.10.0",
```

```suggestion
    "armonik == 3.10.0",
```

Too risky because we may not be backward compatible